test/goss/goss_template.yaml

```diff
@@ -11,6 +11,8 @@ package:
     installed: true
   make:
     installed: true
+  grunt:
```

Comment:
grunt is not a .deb package and its installation can't be tested that way.

Reply:
OK, I delete the check first.